lib/city-state.rb

```diff
@@ -145,6 +145,11 @@ def self.cities(state, country = nil)
     @cities[country][state.to_s.upcase.to_sym] || []
   end
 
+  def self.cities_by_country_name(country_name)
+    country_code = countries.select { |code, name| name.downcase == country_name&.downcase }.keys.first
```

Comment:
Use `detect` instead of `select` & `first`.

Reply:
Hey. Just saw this comment. Thanks 🙂

Reply:
We can also use find.